mysite/static/less/base/base.less

```diff
@@ -83,8 +85,9 @@
 .centered { margin: 0 auto; }
 
 /* styles */
+html { position: relative; min-height: 100%; }
 
-body { font: @normal-size Tuffy, Helvetica, sans-serif; background: @lightest-grey url('/static/images/light-hatch.png') repeat; color: @default-text-color; width: 100%; .centered; padding: 0; }
+body { font: @normal-size Tuffy, Helvetica, sans-serif; background: @lightest-grey url('/static/images/light-hatch.png') repeat; color: @default-text-color; width: 100%; .centered; padding: 0; margin-bottom: @footer-height; }
```

Comment:
I realize conventions are mixed in this file, but since you're changing the line anyways, now seems a great time to break this out of one line.

body {
...
}
 
etc...